src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -158,6 +179,101 @@ private static boolean isKillCountFilterInvalid(int varbitValue) {
         return varbitValue > 0;
     }
 
+    @Synchronized
+    private void exportConfig() {
+        String prefix = CONFIG_GROUP + '.';
+        Map<String, String> configMap = configManager.getConfigurationKeys(prefix)
+            .stream()
+            .map(prop -> prop.substring(prefix.length()))
+            .collect(Collectors.toMap(Function.identity(), key -> configManager.getConfiguration(CONFIG_GROUP, key)));
+        Utils.copyToClipboard(gson.toJson(configMap))
+            .thenRun(() -> plugin.addChatSuccess("Copied current configuration to clipboard"))
+            .exceptionally(e -> {
+                plugin.addChatWarning("Failed to copy config to clipboard");
+                return null;
+            });
+    }
+
+    @Synchronized
+    private void importConfig() {
+        Utils.readClipboard()
+            .thenApplyAsync(json -> {
+                if (json != null) {
+                    try {
+                        return gson.<Map<String, String>>fromJson(json, new TypeToken<Map<String, String>>() {}.getType());
+                    } catch (Exception e) {
+                        String warning = "Failed to parse config from clipboard";
+                        log.warn(warning + ": " + json, e);
```

Comment:
Let's not print the failed JSON here in case someone accidentally tries to import their password